configure

```diff
@@ -196,6 +196,11 @@ shared_optgroup.add_option('--shared-openssl-libpath',
     dest='shared_openssl_libpath',
     help='a directory to search for the shared OpenSSL DLLs')
 
+shared_optgroup.add_option('--use-def-ca-store',
```

Comment:
Why is this in the shared libraries section of the configuration? It should be up with the other openssl options.